app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -196,6 +196,7 @@ class AvatarView @JvmOverloads constructor(
     ) {
         when (avatarDisplayType) {
             AvatarDisplayType.UNKNOWN_CORRESPONDENT -> loadUnknownUserAvatar()
+            AvatarDisplayType.USER_AVATAR -> loadUserAvatar(correspondent!!, bimi)
             AvatarDisplayType.CUSTOM_AVATAR,
             AvatarDisplayType.INITIALS -> loadAvatarUsingDictionary(correspondent!!, contacts, bimi)
             AvatarDisplayType.BIMI -> loadBimiAvatar(bimi!!, correspondent!!)
```

Comment:
This one was actually my fault but maybe swip the parameters so it corresponds to other function.

```suggestion
            AvatarDisplayType.BIMI -> loadBimiAvatar(correspondent!!, bimi!!)
```